client/landing/gutenboarding/lib/analytics/index.ts

```diff
@@ -41,10 +41,12 @@ export function recordOnboardingStart(
 	if ( ! ref ) {
 		ref = new URLSearchParams( window.location.search ).get( 'ref' ) || ref;
 	}
-
-	trackEventWithFlow( 'calypso_newsite_start', { ref, site_count, is_podcasting_site } );
+	const eventProps = is_podcasting_site
+		? { ref, site_count, flow: 'anchor-fm' }
```

Comment:
The `trackEventWithFlow` function accepts a third param with the flow identifier. Is it possible to use it?

Reply:
I initially tried, and didn't see it come through, but I tested it again, and the value is coming through. It might have been perhaps I couldn't find it the first time around.

Thanks!